### PR TITLE
app-laptop/thinkfan version bump to 1.0

### PR DESCRIPTION
**Pull request by tts2k**

### CODE REVIEW

File: app-laptop/thinkfan/thinkfan-1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+inherit cmake-utils readme.gentoo-r1
+
+DESCRIPTION="simple fan control program for thinkpads"
+HOMEPAGE="http://thinkfan.sourceforge.net"
+SRC_URI="https://github.com/vmatare/${PN}/archive/${PV}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="amd64 x86"
```
Comment (Zlogene) [inline app-laptop/thinkfan/thinkfan-1.0.ebuild]: do not commit new versions straight to stable

File: app-laptop/thinkfan/thinkfan-1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+IUSE="atasmart nvidia +yaml"
+
+DEPEND="atasmart? ( dev-libs/libatasmart )
+		yaml? ( dev-cpp/yaml-cpp )"
```
Comment (Zlogene) [inline app-laptop/thinkfan/thinkfan-1.0.ebuild]: one indent please

File: app-laptop/thinkfan/thinkfan-1.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+RDEPEND="${DEPEND}
+		nvidia? ( x11-drivers/nvidia-drivers )"
+
+src_prepare() {
+	cmake-utils_src_prepare
+
+	sed -e "s:share/doc/${PN}:share/doc/${PF}:" \
+		-i CMakeLists.txt
```
Comment (Zlogene) [inline app-laptop/thinkfan/thinkfan-1.0.ebuild]: missing `|| die`

**Comment by Whissi** [comment]
I am merging this but will replace with a fixed ebuild. You missed

- DOCDIR violation (the `sed` wasn't working/doing)
- Bashism in runscript

**Pull request closed without merging**